client/lib/site-roles/test/test-store.js

```diff
@@ -1,52 +1,52 @@
 /**
  * External dependencies
  */
-var assert = require( 'chai' ).assert;
+import { assert } from 'chai';
 
 /**
  * Internal dependencies
  */
-var actions = require( './lib/mock-actions' ),
-	site = require( './lib/mock-site' ),
-	siteId = site.ID;
+import useMockery from 'test/helpers/use-mockery';
 
-require( 'lib/react-test-env-setup' )();
+describe( 'Viewers Store', () => {
+	let Dispatcher, RolesStore;
+	let actions, site, siteId;
 
-describe( 'Viewers Store', function() {
-	var Dispatcher, RolesStore;
+	useMockery();
 
-	beforeEach( function() {
+	before( () => {
 		Dispatcher = require( 'dispatcher' );
 		RolesStore = require( '../store' );
+		actions = require( './mocks/lib/actions' );
+		site = require( './fixtures/site' );
+		siteId = site.ID;
 	} );
 
-	describe( 'Shape of store', function() {
-		it( 'Store should be an object', function() {
+	describe( 'Shape of store', () => {
+		it( 'Store should be an object', () => {
 			assert.isObject( RolesStore );
 		} );
 
-		it( 'Store should have method getRoles', function() {
+		it( 'Store should have method getRoles', () => {
 			assert.isFunction( RolesStore.getRoles );
 		} );
 	} );
 
-	describe( 'Get Roles', function() {
-		it( 'Should return empty object when there are no roles', function() {
-			var roles = RolesStore.getRoles( siteId );
+	describe( 'Get Roles', () => {
+		it( 'Should return empty object when there are no roles', () => {
+			let roles = RolesStore.getRoles( siteId );
```

Comment:
We use `const` for variables that don't change reference. See https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#let-and-const.